ofproto/ofproto.c

```diff
@@ -5664,7 +5664,8 @@ handle_dump_map (struct ofconn *ofconn, const struct ofp_header *oh)
         return OFPERR_OFPBRC_EPERM;
     }
 
-    void *data = NULL;
+    unsigned int map_size = map->ops.map_size(map);
```

Comment:
We need to check that `map->ops.map_size` exists before using it. Same for `map->ops.map_dump` below.

Reply:
Done.